en/after-payment/status-cycle.md

```diff
@@ -2,35 +2,34 @@
-`expire` | Transaction no longer available to be paid or processed, beacause the payment has not been completed after the expiry time period exceeded.
-`failure` | Unexpected error during transaction processing. <br>Failure transaction can be caused by various reasons, but mostly unhandled issue like bank fail to give response (time-out). Occurs rarely.
+`pending` | Transaction is created and available/waiting to be paid by customer at the payment provider (ATM/Internet banking/E-wallet app/store). | settlement, <br>expire,<br>cancel, <br>deny 
+`capture` | Transaction is successful and credit card balance is captured successfully. <br/>If no action is taken by you, the transaction will be successfully settled on the next day and transaction status will change to *settlement*.<br />It is safe to assume a successful payment. | settlement, <br />cancel 
+`settlement` | Transaction is successfully settled. Funds have been received. | refund, chargeback, partial_refund, partial_chargeback
+`deny` | The credentials used for payment are rejected by the payment provider or Midtrans Fraud Detection System (FDS). <br/>To know the reason and details for denied transaction, see the `status_message` field in the response. |
+`cancel` | Transaction is cancelled. Can be triggered by Midtrans or merchant themselves.<br>Cancelled transaction can be caused by various reasons:<br> 1. `Capture` transaction is cancelled before Settlement.<br> 2. `Challenge` Transaction is denied by merchant. |
+`expire` | Transaction no longer available to be paid or processed, because the payment is not completed within the expiry time period. |
+`failure` | The transaction status is caused by unexpected error during transaction processing. <br/>Failure transaction can be caused by various reasons, but mostly it is caused when bank fails to respond.<br/>This occurs rarely. |
 `refund` | Transaction is marked to be refunded. Refund can be requested by Merchant. | 
 `chargeback` | Transaction is marked to be charged back. | 
 `partial_refund` | Transaction is marked to be partially refunded. | 
 `partial_chargeback` | Transaction is marked to be partially charged back. | 
 
-#### Notes When Using Snap API
-<br>
+
 <details>
 <summary><b>Notes When Using Snap API</b></summary>
 <article>
 
 When a transaction is created on Snap API, it does not immediately assign any payment status on Core API's get-status response. 
 
-So please expect that you may encounter `404` or payment not found response upon calling Core API get-status even if the payment page is activated on Snap API. 
+When a transaction is created on Snap API, it does not immediately assign any payment status on *Core API GET Status* response. Even if the payment page is activated on Snap API, you might encounter `404` or *Payment not found* response while calling *Core API GET Status*.
```

Comment:
Looks like you missed to remove this line at the top:
> When a transaction is created on Snap API, it does not immediately assign any payment status on Core API's get-status response.

Because it is duplicated.